CoreFoundation/Preferences.subproj/CFPreferences.h

```diff
@@ -93,7 +93,7 @@ keysToFetch that are not present in the returned dictionary
 are not present in the domain.  If keysToFetch is NULL, all
 keys are fetched. */
 CF_EXPORT
-CFDictionaryRef CFPreferencesCopyMultiple(_Nullable CFArrayRef keysToFetch, CFStringRef applicationID, CFStringRef userName, CFStringRef hostName);
+_Nullable CFDictionaryRef CFPreferencesCopyMultiple(_Nullable CFArrayRef keysToFetch, CFStringRef applicationID, CFStringRef userName, CFStringRef hostName);
```

Comment:
Without this annotation, Swift will reject `(aPref._swiftObject) as? [NSString: Any]` because it doesn't think `_swiftObject` is optional.

I had my doubt then I found functions like `CFPreferencesCopyApplicationList` and `CFPreferencesCopyKeyList` below actually has `_Nullable` annotations.